manifests/2.2.0/opensearch-dashboards-2.2.0.yml

```diff
@@ -9,43 +9,43 @@ ci:
 components:
   - name: OpenSearch-Dashboards
     repository: https://github.com/opensearch-project/OpenSearch-Dashboards.git
-    ref: '2.2'
+    ref: tags/2.2.0
   - name: functionalTestDashboards
     repository: https://github.com/opensearch-project/opensearch-dashboards-functional-test.git
-    ref: '2.2'
+    ref: tags/2.2.0.0
```

Comment:
This should be `2.2.0` with 3 digits version.

Reply:
fixed now. thanks.